cfn/cfn.json

```diff
@@ -321,6 +321,32 @@
                                         "s3:GetObjectAcl"
                                     ],
                                     "Resource": "arn:aws:s3:::gnm-multimedia-rr-deployables/*"
+                                },
```

Comment:
cfn has been moved to the private repo now as it has details on the infrastructure. You'll need to rebase master, remove these changes and issue a PR in the [infra repo](https://github.com/guardian/live-stream-creator-infra).